packages/snaps-simulation/src/structs.ts

```diff
@@ -143,6 +144,22 @@ export const TransactionOptionsStruct = object({
   ),
 });
 
+export const KeyringOptionsStruct = object({
+  /**
+   * The origin making the Keyring request.
+   */
+  origin: defaulted(string(), 'metamask.io'),
+  /**
+   * Keyring request object with params.
+   */
+  request: object({
```

Comment:
Should this mirror `RequestOptions` and not have a separate request object?

cc @Mrtenz

Reply:
Refactored to use `RequestOptions`.